package.json

```diff
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "dev": "webpack -d --progress --profile --colors --bail --",
-    "serve": "webpack-dev-server -d --inline --https --host 0.0.0.0 --content-base troposphere/assets",
+    "serve": "webpack-dev-server -d --inline --https --content-base troposphere/assets",
```

Comment:
Since I'm guilty of not using the `webpack-dev-server` option right, I wanted to ask about why the the catch-all `0.0.0.0` host is removed?